contracts/protocol/libs/NativeClaims.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+/**
+ * @title Native Claims Storage Library
+ * @notice Provides storage for native currency claims that can be used by both protocol and client contracts
+ */
+library NativeClaims {
+    bytes32 private constant STORAGE_SLOT = keccak256("native.claims.storage");
```

Comment:
Let's use ERC-7201 for storage location as we do elsewhere.

Also, precompute it, so you then don't have to introduce `bytes32 slot` in `_getStorage()`. (Again, refer to other places where we deal with storage how it's done.).
I think the slot is `0xaf833ba755a88e67fdf7592ad2cd23cb88eeb39ba7091cbf219ca880c31f2a00` but check it yourself.